src/morphology_workflows/curation.py

```diff
@@ -461,14 +460,33 @@ def orient(row, data_dir, pia_direction="y"):
     return ValidationResult(is_valid=True, morph_path=new_morph_path)
 
 
-def align(row, data_dir, method="whole", neurite_type="apical", direction=None):
+def align(
```

Comment:
There are many lines that are not covered in this function, it would be nice to improve its coverage with a new test.

Reply:
ah, this is because align is skipped by default. Maybe we should make a test run for all the skipped tasks in another PR?

Reply:
Good idea!